### PR TITLE
feat(frontend): account page

### PR DESCRIPTION
**Pull request by cristian-ramirez**
### Account page #90

#

### Steps to test

1. Create account in [jungle](https://monitor.jungletestnet.io/#home).
1. Set up the credentials in local Scatter.
1. Open up the app in desktop.
1. Click in the fingerprint icon in the right top corner.

### Checklist
- [ ] I have added/updated tests to cover these changes.
- [x] The branch name, commit history and PR title follow [conventions](https://developers.eoscostarica.io/docs/open-source-guidelines#pull-request-general-guidelines).
- [x] I have taken into consideration any impact to site performance.

#

account_page

### CODE REVIEW

**Comment by ramijames** [comment]
Guys! Stop using the chrome extension! :)

File: services/frontend/src/services/scatter.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ import ScatterEOS from 'scatterjs-plugin-eosjs2'
...
 const network = {
   blockchain: 'eos',
-  protocol: 'https',
-  host: 'nodes.get-scatter.com',
-  port: 443,
-  chainId: 'aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906'
+  protocol: 'http',
+  host: 'jungle2.cryptolions.io',
+  port: 80,
+  chainId: 'e70aaab8997e1dfce58fbfac80cbbb8fecec7b99cf982a9444273cbc64c41473'
```
Comment (arturocr) [inline services/frontend/src/services/scatter.js]: Do we want to merge these config changes to master? I applied something similar locally for dev purposes, but if we merge this it means that the login will be performed against Jungle Testnet.
@gaboesquivel, @danazkari can you give us your input on this?

**Comment by arturocr** [comment]
Works locally for me:
EOSRate-Account

Let's wait for Gabo and Prado's input on [comment above](https://github.com/eoscostarica/eos-rate/pull/165#discussion_r264963317).

File: services/frontend/src/services/scatter.js (hunk 1 above)
Comment (gaboesquivel) [reply services/frontend/src/services/scatter.js]: Yes, Jungle

**Comment by cristian-ramirez** [comment]
I'll update the width in order to avoid broken.

File: services/frontend/src/services/scatter.js (hunk 1 above)
Comment (arturocr) [reply services/frontend/src/services/scatter.js]: Perfect then, thanks.

**Comment by arturocr** [comment]
> I'll update the width in order to avoid broken.

Sweet, I already approved this.

**Pull request merged**